src/main/java/lk/gov/health/phsp/entity/Client.java

```diff
@@ -210,20 +207,15 @@ public String getPhn() {
     public void setPhn(String phn) {
         this.phn = phn;
     }
-    
-    
 
 // </editor-fold>
+    public List<ClientEncounterComponentItem> getClientEncounterComponentItems() {

```

Comment:
Need to remove getters and setters as well

Reply:
Done